troop/forms.py

```diff
@@ -52,3 +55,7 @@ class Meta:
             "diet": CheckboxSelectMultiple,
             "comment": Textarea(attrs={"cols": 32, "rows": 4}),
         }
+
+
+class NamiSearchForm(Form):
```

Comment:
You are right, if we strictly follow the meaning of `NaMi`, but I find it way more readable to be `Nami`.
If we go with `NaMi*` everywhere, then we should also replace `nami` with `na_mi` everywhere to be consistent (which really hurts readability I think).

The URL should also be `participant/na-mi-search`

What do you think?

Reply:
I'd use `NaMi` and `nami`. I think it's okay to skip the separator in the lowercase version (except you want to write `namentliche-mitgliedermeldung`). But `Nami` really hurts my eyes, it conveys that `Nami` is a word.

Reply:
Apparently a lot of people prefer `Nami*` for code:
- https://github.com/webratz/pynami/blob/master/pynami/nami.py#L37
- https://github.com/DaSchaef/NBAS/blob/master/classes/NamiConnection_class.php
- https://github.com/platdesign/node-nami-api-client/blob/master/index.js#L33
- https://github.com/fabianlipp/jnami/blob/master/src/nami/nami2mailman/Nami2Mailman.java#L31

Reply:
I don't know why. It doesn't make any sense when using camel case. Since when are some initials of words lowercase in camel case? And `nami` is just too short to put a dash into.

Reply:
I personally prefer `Nami` much more (it is just too short to use camel case inside ;-)

Reply:
Ok :man_shrugging: